hw02/hw02_exploring_gapminder.Rmd

```diff
@@ -0,0 +1,65 @@
+library(gapminder)
+library(ggplot2)
+library(palmerpenguins)
+library(tidyverse)
+library(dplyr)
+
+# 1.1 & 1.2
+
+gapminder %>%
+  select(country, year, gdpPercap) %>%
+  filter(country %in% c("Peru" , "Japan", "Canada"), year>1969 & year<1980)
+
+# 1.3
+
+gapminder %>%
+  mutate(lifeExpDrop = lag(lifeExp)
```

Comment:
It seems like this line of code is incomplete. I recommend rechecking your formula and don't forget to filter the database!